Incompatible version scheme 0.0.x

The current version is `0.0.12` which according to [Specifying Dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html):

> The version 0.0.x is not considered compatible with any other version.

This appears to be causing an issue when installing wash-cli because the transitive dependencies for crates like `wascap` get updated containing potentially new versions of nkeys, but the direct nkeys dependency will stay the same causing errors like this later as new versions get published:

```
error[E0308]: mismatched types
   --> src/claims.rs:367:34
    |
367 |     println!("{}", claims.encode(&issuer)?);
    |                                  ^^^^^^^ expected struct `wascap::prelude::KeyPair`, found struct `nkeys::KeyPair`
    |
    = note: expected reference `&wascap::prelude::KeyPair`
               found reference `&nkeys::KeyPair`
    = note: perhaps two different versions of crate `nkeys` are being used?
```
I believe this can be alleviated by incrementing the nkeys version to `0.1.0` so that any compatible versions for direct and transitive dependencies will get updated if possible at the same time, avoiding the mismatched types problem.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "nkeys"
-version = "0.0.12"
+version = "0.1.0"
 authors = ["Kevin Hoffman <alothien@gmail.com>"]
 edition = "2018"
 description = "Rust implementation of the NATS nkeys library"

